### ISSUE TITLE
Version of dexmaker included here, conflicts with dexmaker in my tests.

### ISSUE THREAD

**Issue by yogurtearl** [facebook/screenshot-tests-for-android#11]
screenshot-tests-for-android includes ```com.google.dexmaker```
my tests use ```com.crittercism.dexmaker``` . 

When applying the screenshot-tests-for-android plugin, about 40% of the time some of my tests that use Mockito will fail with error like this:

```java.lang.IllegalArgumentException
    at com.google.dexmaker.mockito.InvocationHandlerAdapter.invoke(InvocationHandlerAdapter.java:49)
    at java.lang.reflect.Proxy.invoke(Proxy.java:397)
    at $Proxy33.clearOffersCache(Unknown Source)
    at ```

and 

```org.mockito.exceptions.misusing.UnfinishedVerificationException: 
Missing method call for verify(mock) here:
-> at 
Example of correct verification:
    verify(mock).doSomething()

Also, this error might show up because you verify either of: final/private/equals()/hashCode() methods.
Those methods *cannot* be stubbed/verified.
Mocking methods declared on non-public parent classes is not supported.
```

I can tell it is related to the ```com.google.dexmaker``` dependancy, because of ```com.google.dexmaker.mockito.InvocationHandlerAdapter``` in the stacktrace.

```com.crittercism.dexmaker``` uses a different package name -> "com.android.dx.mockito"

**Comment by tdrhq** [comment]
So do I understand correctly the that's the new package name for dexmaker? 

So basically I'm using an old version, and I should be able to drop in the new version, right?

**Comment by yogurtearl** [comment]
new dexmaker uses ```com.android.dx``` as the java package

new dependencies:

```
com.crittercism.dexmaker:dexmaker:1.4
com.crittercism.dexmaker:dexmaker-mockito:1.4
com.crittercism.dexmaker:dexmaker-dx:1.4
```

**Comment by josephmbeveridge** [comment]
Same issue here.

I am getting `AbstractMethodError` on non-abstract methods.

**Comment by tdrhq** [comment]
Upgraded to 1.4 d1e5756f0272fad6711200430c627a3a9ac63721 and released a new version with v0.2.4

There are some other changes in this release. Please let me know if this works, and I'll close the task.

**Comment by josephmbeveridge** [comment]
This has fixed the issue for me! :+1: Thanks for the quick release
